queries.sql

```diff
@@ -370,3 +370,8 @@ LIMIT 1;
 -----------------------------------------------------------------
  -- Digimon
 -- (1 row)
+
+/* Performance Audit */
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animal_id = 4;
+EXPLAIN ANALYZE SELECT * FROM visits where vet_id = 2;
```

Comment:
* Kindly improve the execution time for this query as the execution time on the PR screenshot can be lower

Reply:
Do you have any suggestions on how to lower the executions time? We added the index on the correct column (vets_id) and the execution time did go down by a bit (3546ms -> 3286ms). Not sure how we can make it go lower still.

Reply:
DONE